src/main/resources/org/jenkinsci/plugins/ewm/nodes/NodeDiskPool/config.jelly

```diff
@@ -6,4 +6,9 @@
     <f:entry>
         <f:repeatableProperty field="nodeDisks" add="${%Add Disk}" header="${%Disk}"/>
     </f:entry>
+    <f:entry>
+        <div align="right">
+            <f:repeatableDeleteButton value="${%Delete Disk Pool}"/>
```

Comment:
If I understand correctly, the calling jelly for NodeDiskPool/config.jelly is Template/config.jelly. When I move the delete button to the Template/config.jelly locally, the Disk Pool in the Node Configuration is missing the delete button again.